components/collective-page/sections/Budget.js

```diff
@@ -40,8 +41,16 @@ export const budgetSectionQuery = gqlV2/* GraphQL */ `
   ${expensesListFieldsFragment}
 `;
 
+const DEFAULT_KINDS = [
+  TransactionKind.ADDED_FUNDS,
+  TransactionKind.CONTRIBUTION,
+  TransactionKind.EXPENSE,
+  TransactionKind.PLATFORM_TIP,
```

Comment:
It's a special case but in the context of Open Collective budget, that will be noisy to have platform tips there.

Reply:
It's a bit of work to adapt the conditions to make sure the cache will still work properly in this case. Since we already display them on OC's budget since their introduction, I'm rater opening a follow-up issue for this one.

Reply:
=> https://github.com/opencollective/opencollective/issues/4414